model/pod/pod.go

```diff
@@ -0,0 +1,74 @@
+package pod
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// Pod represents a related collection of containers. This model is hold
+// metadata about containers running in a container orchestration system.
+type Pod struct {
+	// ID is the unique identifier for the metadata document. This is
+	// semantically unrelated to the PodID.
+	ID string `bson:"_id" json:"id"`
+	// PodID is the unique resource identifier for the collection of containers
+	// running in the container orchestration service.
+	PodID string `bson:"pod_id" json:"pod_id"`
+	// TaskCreationOpts are options to configure how a task should be
+	// containerized and run in a pod.
+	TaskContainerCreationOpts TaskContainerCreationOptions `bson:"task_creation_opts,omitempty" json:"task_creation_opts,omitempty"`
+	// TimeInfo contains timing information for the pod's lifecycle.
+	TimeInfo TimeInfo `bson:"time_info" json:"time_info"`
+}
+
+// TaskCreationOptions are options to will apply to the task's container when
+// creating a pod in the container orchestration service.
+type TaskContainerCreationOptions struct {
+	// Image is the image that the task's container will run.
+	Image string `bson:"image" json:"image"`
+	// MemoryMB is the memory (in MB) that the task's container will be
+	// allocated.
+	MemoryMB int `bson:"memory_mb" json:"memory_mb"`
+	// CPU is the CPU units that the task will be allocated. 1024 CPU units is
+	// equivalent to 1vCPU.
+	CPU int `bson:"cpu" json:"cpu"`
+	// IsWindows indicates whether or not the task will run in a Windows
```

Comment:
`IsWindowsContainer`, though, maybe `IsWindows` is more concise and still clear?

Reply:
Sure, I'm fine with `IsWindows`.